skimage/measure/__init__.py

```diff
@@ -7,7 +7,7 @@
 from ._pnpoly import points_in_poly, grid_points_in_poly
 from ._moments import moments, moments_central, moments_normalized, moments_hu
 from .profile import profile_line
-from .fit import LineModel, CircleModel, EllipseModel, ransac
+from .fit import LineModel, LineModel3D, CircleModel, EllipseModel, ransac
```

Comment:
I'm not intimately familiar with the underlying mechanisms with passing these models through RANSAC. However, it seems the ideal case would be to intuit from the input data which line is being fit, and automatically use the appropriate model. Thus no deprecation, with users just employing `LineModel`.

Is that possible? It would probably be the most ideal API.